CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ add_definitions(${thirdparty_def})
 
 if(MBEDTLS)
 
-	set(T_COSE_SOURCE crypto_adapters/t_cose_psa_crypto.c src/t_cose_sign1_sign.c src/t_cose_parameters.c src/t_cose_sign1_verify.c src/t_cose_util.c)
+	set(T_COSE_SOURCE crypto_adapters/t_cose_psa_crypto.c src/t_cose_sign1_sign.c src/t_cose_parameters.c src/t_cose_sign1_verify.c src/t_cose_util.c  src/t_cose_encrypt_enc.c src/t_cose_encrypt_dec.c)
```

Comment:
This is getting long enough that it might make sense to break this into multiple lines.